src/fidesops/ops/service/privacy_request/request_runner_service.py

```diff
@@ -343,20 +354,70 @@ async def run_privacy_request(
         )
         if not proceed:
             return
-
+        if config.notifications.send_request_completion_notification:
+            try:
+                initiate_privacy_request_completion_email(
+                    session, policy, access_result_urls, identity_data
+                )
+            except IdentityNotFoundException as e:
```

Comment:
If there was no email supplied originally, even if we set it up so a fidesops admin can restart this privacy request (by caching a checkpoint), it doesn't change the fact that we don't have an email at all.  These will just stay in a failure state I assume? It seems like there's no way to correct.

Reply:
Right, so this is the edge case where, for example,  the original identity input would be only a phone number (no email), and so the email notification cannot send. Right now, this shouldn't ever happen, since we always require an email in privacy center. But it's possible a user could build their own privacy center and call fidesops with only a phone number. 

My thinking is at least adding to docs that it's required for a privacy center to intake email identity for email notifications to function properly.